src/Microsoft.Diagnostics.Monitoring.RestServer/Models/LogsConfiguration.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Extensions.Logging;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.Diagnostics.Monitoring.RestServer.Models
+{
+    public class LogsConfiguration
+    {
+        [JsonPropertyName("logLevel")]
+        [EnumDataType(typeof(LogLevel))]
+        [Required]
```

Comment:
Shouldn't level be optional? It's not really needed if we useAppfilters

Reply:
It's require and has a default value. This effectively prevents someone from stating:
```json
{
    "logLevel": null
}
```
I don't think this is really a valid scenario. Also, it prevents the following:
```json
{
    "filterSpecs": {
        "Microsoft.AspNetCore.Hosting": null
    },
    "logLevel": null
}
```
Which is most definitely not allowed. Because the category has a null level, it would fallback to the default log level (which is also null).

We cannot use null when setting up the EventPipeProvider. Just based on how event source works, the log level (which is converted to event level) is required.

We could allow for nullability and then provide a default value if it is null, but that's not obvious what that value will be (maybe `LogLevel.Trace`/`EventLevel.Verbose`)

Reply:
I suppose that since it is not a nullable property, then I can remove the Required attribute.